test/controllers/graphql_controller_v2_test.rb

```diff
@@ -632,36 +632,11 @@ def setup
       # TIMDEX will throw an error for non-nullable subfields if the parent field is null, so we only need to find null
       # values of 'holdings' to test this.
       assert json['data']['search']['records'].map { |record| record['holdings'].nil? }.any?
-      assert_not json['errors'].present?
+      refute json['errors'].present?
     end
   end
 
-  test 'graphqlv2 can retrieve geolocation information' do
-    VCR.use_cassette('graphql v2 geolocation') do
-      post '/graphql', params: { query: '{
-                                  search(searchterm: "train stations") {
-                                    records {
-                                      locations {
-                                        geopoint
-                                        geoshape
-                                        kind
-                                        value
-                                      }
-                                    }
-                                  }
-                                }' }
-      assert_equal(200, response.status)
-      json = JSON.parse(response.body)
-
-      assert_not json['errors'].present?
-      assert_equal(
-        json['data']['search']['records'].first['locations'][0].keys.sort,
-        ["geopoint", "geoshape", "value", "kind"].sort
-      )
-    end
-  end
```

Comment:
Fair enough, yeah - I was iffy on adding this test anyway.

Reply:
I actually don't think I meant to delete this test. At some point I rebased after your code merged and somehow I lost this in the process. I'll reintroduce it.